components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/internal/FrameworkServiceComponent.java

```diff
@@ -775,4 +778,15 @@ private AuthenticatorConfig getAuthenticatorConfig(String name) {
         }
         return authConfig;
     }
+
+    private JsGraphBuilderFactory createJsGraphBuilderFactory() {
+        String scriptEngineName = IdentityUtil.getProperty("AdaptiveAuth.ScriptEngine");
```

Comment:
let's move to constant these to `AdaptiveAuth.ScriptEngine` and `graaljs`

Reply:
Modified

Reply:
Modified in all related scenarios